api/envoy/type/v3/ratelimit_unit.proto

```diff
@@ -0,0 +1,30 @@
+syntax = "proto3";
+
+package envoy.type.v3;
+
+import "udpa/annotations/status.proto";
+
+option java_package = "io.envoyproxy.envoy.type.v3";
+option java_outer_classname = "RatelimitUnitProto";
+option java_multiple_files = true;
+option (udpa.annotations.file_status).package_version_status = ACTIVE;
+
+// [#protodoc-title: Ratelimit Time Unit]
+
+// Identifies the unit of of time for rate limit.
+enum RateLimitUnit {
```

Comment:
This is copied out from rls.proto, right? Can we at least put a note in that proto to use this type in the next major version? I think I would also be OK with a compile breaking (not wire) change to just use this new proto for the RLS service but will defer to @htuch on that.

Reply:
Yeah. I wanted to reuse it there initially and then realized that would not be b/c. Left a TODO comment for v4 in rls.proto